sirius/features/org.polarsys.kitalpha.sirius.typeinterpreter.feature/feature.properties

```diff
@@ -0,0 +1,181 @@
+##
+#  Copyright (c) 2021 Thales Corporate Services S.A.S.
```

Comment:
Can you replace references of THALES CORPORATE SERVICES with the usual copyright in Kitalpha, 

https://github.com/eclipse/kitalpha/blob/master/cadence/plugins/org.polarsys.kitalpha.cadence.ui/src/org/polarsys/kitalpha/cadence/ui/internal/CadenceHelper.java#L1-L11